src/main/kotlin/com/sourcegraph/cody/inspections/CodyFixHighlightPass.kt

```diff
@@ -97,6 +101,12 @@ class CodyFixHighlightPass(val file: PsiFile, val editor: Editor) :
                 null
               }
             }
+            .toList()
+
+    if (protocolDiagnostics.isEmpty()) {
```

Comment:
it was possible that we have been sending an empty list 😅 that was completely redundant